packages/frontend/src/features/sqlRunner/components/Sidebar.tsx

```diff
@@ -75,8 +74,10 @@ export const Sidebar: FC<Props> = ({ setSidebarOpen }) => {
                 }
                 sx={{ flex: 1, overflow: 'hidden' }}
             >
-                <LoadingOverlay visible={isLoading} />
-                <TablesPanel />
+                <TablesPanel
+                    isLoading={isLoading}
+                    error={error?.message || null}
```

Comment:
then here we can rewrite the message accessing:
```suggestion
                 error={error?.error.message || null}
```

Reply:
Something to consider in the future: we could just have a generic error message to the user if there is an error, like `Couldn't get tables (or wtv)`